frontend/src/lib/components/common/badge/Badge.svelte

```diff
@@ -0,0 +1,92 @@
+<script lang="ts">
+	import { classNames } from '$lib/utils'
+	import { CloseButton } from 'flowbite-svelte'
+	import Icon from 'svelte-awesome'
+	import { type BadgeColor, type BadgeIconProps, ColorModifier } from './model'
+
+	export let color: BadgeColor = 'gray'
+	export let large = false
+	export let href = ''
+	export let rounded = false
+	export let index = false
+	export let dismissable = false
+	export let baseClass = 'text-center -mb-0.5'
+	export let icon: BadgeIconProps | undefined = undefined
+
+	let defaulIconProps: BadgeIconProps = {
+		data: undefined,
+		position: 'left',
+		scale: 0.7
+	}
+	let hidden = false
+	const colors: Record<BadgeColor, string> = {
+		gray: 'bg-gray-100 text-gray-800 dark:bg-gray-700 dark:text-gray-300',
```

Comment:
As a rule a thumb, please add as little code as necessary and we'll add more colors if we need them later.